rollup.config.js

```diff
@@ -7,6 +7,22 @@ import replace from '@rollup/plugin-replace'
 import nodeResolve from '@rollup/plugin-node-resolve'
 import commonjs from '@rollup/plugin-commonjs'
 
+function getEnv(key, defaultValue) {
```

Comment:
Could we use the refactored version of this function? It can be found [here](https://github.com/fingerprintjs/fingerprint-pro-azure-integration/blob/rc/rollup.config.mjs#L16). This version is more readable.